src/Chess.cpp

```diff
@@ -0,0 +1,257 @@
+	#include "Chess.h"
+
+// clear the screen "cls"
+void Chess::clear() const 
+{
+	COORD topLeft = { 0, 0 };
+	HANDLE console = GetStdHandle(STD_OUTPUT_HANDLE);
+	CONSOLE_SCREEN_BUFFER_INFO screen;
+	DWORD written;
+
+	GetConsoleScreenBufferInfo(console, &screen);
+	FillConsoleOutputCharacterA(
+		console, ' ', screen.dwSize.X * screen.dwSize.Y, topLeft, &written
+	);
+	FillConsoleOutputAttribute(
+		console, FOREGROUND_GREEN | FOREGROUND_RED | FOREGROUND_BLUE,
+		screen.dwSize.X * screen.dwSize.Y, topLeft, &written
+	);
+	SetConsoleCursorPosition(console, topLeft);
+}
+// create the GUI - ASCII art
+void Chess::setFrames() 
+{ 
+	// set all to ' ' (space bar ascii value 32) instead of 0  
+	{
+		for (size_t row = 0; row < _SIZE; ++row)
+			for (size_t col = 0; col < _SIZE; ++col)
+				m_board[row][col] = 32;
```

Comment:
Use constants: Instead of hard-coding ASCII values throughout the code, define constants with meaningful names to improve code readability. For example, instead of using 32 directly, define a constant like SPACE_ASCII = 32.